src/uwtools/template.py

```diff
@@ -46,11 +46,11 @@ class Template:
     """
 
     substitutions = {
-        TemplateConstants.DOLLAR_CURLY_BRACE: TemplateConstants.SubPair(re.compile('\${.*?}+'), slice(2, -1)),
-        TemplateConstants.DOLLAR_PARENTHESES: TemplateConstants.SubPair(re.compile('\$\(.*?\)+'), slice(2, -1)),
-        TemplateConstants.DOUBLE_CURLY_BRACES: TemplateConstants.SubPair(re.compile('{{.*?}}+'), slice(2, -2)),
-        TemplateConstants.AT_SQUARE_BRACES: TemplateConstants.SubPair(re.compile('@\[.*?\]+'), slice(2, -1)),
-        TemplateConstants.AT_ANGLE_BRACKETS: TemplateConstants.SubPair(re.compile('@\<.*?\>+'), slice(2, -1))
+        TemplateConstants.DOLLAR_CURLY_BRACE: TemplateConstants.SubPair(re.compile(r'\${.*?}+'), slice(2, -1)),
```

Comment:
Good catch.

Reply:
@ "good catch" Yes if we are referring to the usefulness of this encapsulation where the slice to access the KEY associated with the abstracted name in CAPS along with the regular expression then I'll mark this conversation as closed.